modules/database/sqlite/wp-includes/sqlite/class-wp-sqlite-query-rewriter.php

```diff
@@ -0,0 +1,343 @@
+<?php
+/**
+ * Class WP_SQLite_Query_Rewriter
+ *
+ * @package performance-lab
+ */
+
+/**
+ * The query rewriter class.
+ */
+class WP_SQLite_Query_Rewriter {
```

Comment:
Same comment above applies here and elsewhere on the other classes introduced here. Shouldn't we use a safer plugin-specific prefix for our code?